src/main/java/bio/terra/service/filedata/DrsService.java

```diff
@@ -155,6 +159,38 @@ public void close() {
     }
   }
 
+  /**
+   * Determine the acceptable means of authentication for a given DRS ID, including the passport
+   * issuers when supported.
+   *
+   * @param drsObjectId the object ID for which to look up authorizations
+   * @return the `DrsAuthorizations` for this ID
+   * @throws IllegalArgumentException if there is an issue with the object id
+   * @throws SnapshotNotFoundException if the snapshot for the DRS object cannot be found
+   * @throws TooManyRequestsException if there are too many concurrent DRS lookup requests
+   */
+  public DRSAuthorizations lookupAuthorizationsByDrsId(String drsObjectId) {
+    try (DrsRequestResource r = new DrsRequestResource()) {
+      DRSAuthorizations auths = new DRSAuthorizations();
+
+      auths.addSupportedTypesItem(DRSAuthorizations.SupportedTypesEnum.BEARERAUTH);
+      // TODO: add to bearer_auth_issuers, ask Muscles.
```

Comment:
I think we want to leave this blank here.  @dvoet : just want to check is the idea here to add allowed oauth audiences?  If so, I don't think we can infer that.  We'd need to add options to set these at the dataset/snapshot level (in that case, it's certainly beyond the scope of this PR).